components/shared/searchFilters/SearchFiltersColumn.tsx

```diff
@@ -48,42 +50,61 @@ const SearchFiltersColumn = ({
     <>
       <div
         key={facet.name}
-        className={cn("relative ml-[-4px]", !isLast && "min-w-32 flex-1", isLast && "flex-2")}>
-        <h3 className="mb-2 pl-2 text-typo-caption uppercase">
-          {getFacetTranslation(facetFilter)}
-        </h3>
-        <div
-          className={cn(
-            "flex gap-1 px-1 pt-2 text-typo-caption",
-            !isLast && "flex-col",
-            isLast && "flex-row flex-wrap content-start",
-            !isExpanded && "h-[107px] overflow-hidden"
-          )}
-          ref={elementRef}>
-          {facet.values.map((value, index) => (
+        className={cn(
+          "space-y-grid-gap-half relative",
+          !isLast && "min-w-32 flex-1",
+          isLast && "flex-2"
+        )}>
+        <h3 className="text-typo-caption uppercase">{getFacetTranslation(facetFilter)}</h3>
+
+        <AnimateChangeInHeight className="overflow-visible">
+          <div
+            className={cn(
+              "mx-[-10px] mt-[-10px] flex gap-1 px-[10px] pt-[10px] text-typo-caption",
+              !isLast && "flex-col",
+              isLast && "flex-row flex-wrap content-start",
+              !isExpanded && "h-[102px] overflow-hidden"
+            )}
```

Comment:
```suggestion
           className={cn(
              "mx-[-10px] mt-[-10px] flex gap-1 px-[10px] pt-[10px] text-typo-caption",
              {
                "flex-row flex-wrap content-start": isLast,
                "flex-col": !isLast,
                "h-[102px] overflow-hidden": !isExpanded,
              }
            )}
```

Reply:
You get my point :) Consider, if you agree,  using that pattern for conditionals in clsx